Documentation/Syntax/StringFormats/Index.rst

```diff
@@ -0,0 +1,175 @@
+:navigation-title: String formats
+..  include:: /Includes.rst.txt
+..  _data-types:
+..  _data-types-reference:
+..  _string-formats:
+
+=========================
+TypoScript string formats
+=========================
+
+The values assigned to properties in TypoScript are internally handled as
+strings.
+
+Some properties expect a certain format of string. These formats are described
+in this chapter.
+
+Enforcing these string formats is up to the implementation of the property using
+them. They are therefore no real constructs of the TypoScript syntax.
+
+..  contents::
+    :local:
+
+..  index:: Simple data types; boolean
+..  _data-type-boolean:
+
+boolean
+=======
+
+..  confval:: boolean
+    :name: data-type-boolean
+
+    Possible values for boolean variables are `1` and `0`
+    meaning TRUE and FALSE.
+
+    Everything else is `evaluated to one of these values by PHP`__:
+    Non-empty strings (except `0` [zero]) are treated as TRUE,
+    empty strings are evaluated to FALSE.
+
+    __ https://www.php.net/manual/en/language.types.boolean.php
+
+    ..  rubric:: Examples
+
+    .. code-block:: typoscript
+       :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+
+       dummy.enable = 0   # false, preferred notation
+       dummy.enable = 1   # true,  preferred notation
+       dummy.enable =     # false, because the value is empty
```

Comment:
```suggestion
    ..  code-block:: typoscript
        :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript

        dummy.enable = 0   # false, preferred notation
        dummy.enable = 1   # true,  preferred notation
        dummy.enable =     # false, because the value is empty
```